Puppetfile

```diff
@@ -6,13 +6,13 @@ moduledir File.join(File.dirname(__FILE__), 'modules')
 
```

Comment:
Looks like the commit history git a bit messed up. You'll need to rebase off the `main` branch of the upstream repo so [this commit](https://github.com/puppetlabs/bolt/pull/2914/commits/ffbbfc2e2eb64e376a98e0ea343b8c7b8fd1d3e1) isn't part of the history.